src/cpp/server/server_context.cc

```diff
@@ -157,14 +157,15 @@ class BackendMetricState : public experimental::CallMetricRecorder,
   }
 
  private:
+  // Once returned metrics recorded with CallMetricRecorder APIs are cleared.
```

Comment:
I don't understand this comment.

Reply:
Re-worded. I wanted to note that calling this twice won't produce the same result as this now clears internal metrics (moves maps).